MaxText/configs/1xv4-384.sh

```diff
@@ -14,5 +14,5 @@
 
 export LIBTPU_INIT_ARGS="--xla_enable_async_all_gather=true TPU_MEGACORE=MEGACORE_DENSE"
 RUN_NAME=1x_v4-384_0013
-python3 MaxText/train.py MaxText/configs/base.yml run_name=${RUN_NAME} enable_profiler=true enable_checkpointing=false steps=10 ici_fsdp_parallelism=192 ici_tensor_parallelism=1 scale=4 base_num_decoder_layers=8 per_device_batch_size=10 remat_policy=full base_emb_dim=3072 base_mlp_dim=12288 use_pjrt=0  learning_rate=1e-8
```

Comment:
It should be fine to always set it true.

Maxtext will throw an error if we try to set it all as a config parameter in these .sh files since it no longer exists in base.yml. It can be overriden by users if they include an export command before their train.py, but I see no reason for anyone to do that.

Reply:
Gotcha, thanks for the info!